strings.json

```diff
@@ -47,52 +58,69 @@
     "en": "Omit Cash from Percentage Returns"
   },
   "accessibility_realizedChange": {
-    "en": "Realized Change"
+    "en": "Realized Change",
+    "zh-Hant-TW": "已實現損益"
   },
   "accessibility_realizedChangePercent": {
-    "en": "Realized Change Percent"
+    "en": "Realized Change Percent",
+    "zh-Hant-TW": "已實現獲利率"
   },
   "accessibility_realizedYTDChange": {
-    "en": "Realized YTD Change"
+    "en": "Realized YTD Change",
+    "zh-Hant-TW": "已實現年化報酬"
   },
   "accessibility_realizedYTDChangePercent": {
-    "en": "Realized YTD Change Percent"
+    "en": "Realized YTD Change Percent",
+    "zh-Hant-TW": "已實現年化報酬率"
   },
   "accessibility_realizedCostBasis": {
-    "en": "Realized Cost Basis"
+    "en": "Realized Cost Basis",
+    "zh-Hant-TW": "投資成本"
   },
   "accessibility_realizedValue": {
-    "en": "Realized Value"
+    "en": "Realized Value",
+    "zh-Hant-TW": "帳面收入"
   },
   "accessibility_refreshNews": {
-    "en": "Refresh News"
+    "en": "Refresh News",
+    "zh-Hant-TW": "重新整理"
```

Comment:
Is '重新整理' the right term for 'Refresh News'? Refresh news means to refresh/reload the page to see if there's newer news. Does '重新整理' mean 'Reorganize'?

Reply:
What's difference between refresh and reload
Refresh could be "重新整理","重新載入" or "刷新".
It might not needy to translate "news"

Reply:
Since this is "accessibility_refreshNews", it means it's for accessibility (people who can't see clearly). So it's better to be a bit more descriptive and say that the refresh is for refreshing news